src/test/scala/mesosphere/marathon/api/akkahttp/v2/PodsControllerTest.scala

```diff
@@ -55,7 +59,12 @@ class PodsControllerTest extends UnitTest with ScalatestRouteTest with RouteBeha
       val request = Post(Uri./.withQuery(Query("force" -> "false")))
         .withEntity(entity)
         .withHeaders(`Remote-Address`(RemoteAddress(InetAddress.getByName("192.168.3.12"))))
+
+      val podDefinition = PodDefinition(id = PathId("mypod"))
+      f.podManager.find(any).returns(Some(podDefinition))
```

Comment:
Can we explicitly set the id here? For example: `f.podManager.find(equals(id)).returns(Some(podDefinition))`

Reply:
Good point.

Reply:
✅

Reply:
Here we do not care what parameters are passed. We test the authorization for a pod.